src/coreclr/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -858,6 +858,12 @@ private TypeSystemEntity entityFromContext(CORINFO_CONTEXT_STRUCT* contextStruct
             return (TypeSystemEntity)HandleToObject((IntPtr)((ulong)contextStruct & ~(ulong)CorInfoContextFlags.CORINFO_CONTEXTFLAGS_MASK));
         }
 
+        private bool isJitIntrinsic(CORINFO_METHOD_STRUCT_* ftn)
+        {
+            MethodDesc method = HandleToObject(ftn);
+            return method.IsIntrinsic;
```

Comment:
I think the only particular thing of note is this (as per https://github.com/dotnet/runtime/compare/main...tannergooding:isJitIntrinsic?expand=1#diff-132a77bcd3f74cf0e0b04fbccda246c97c91e40562d78cb01fff61cf69403573L872-L873) isn't fine grained enough to determine just `CORINFO_FLG_INTRINSIC` or `CORINFO_FLG_JIT_INTRINSIC`